docs/src/lib/Code.svelte

```diff
@@ -11,17 +11,17 @@
   hljs.registerLanguage("plaintext", plaintext);
 </script>
 
-<script>
+<script lang="ts">
   import "highlight.js/styles/github.css";
 
-  /** @param {string} markup */
-  const trim = (markup) => markup.replace(/^(\s*\n)+/, "").replace(/\n\s*$/, "");
+  interface Props {
+    language?: "html" | "css" | "scss" | "plaintext";
+    code?: string;
+  }
 
-  /** @type {'html' | 'css' | 'scss' | 'plaintext'} */
-  export let language = "plaintext";
-  export let code = "";
-  $: trimmed = trim(code);
-  $: highlighted = hljs.highlight(trimmed, { language }).value;
+  let { language = "plaintext", code = "" }: Props = $props();
```

Comment:
I was just wondering why not using the `export let ...` when I saw the documentation about Svelte 5. Nice.